### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: components/nexus-repository/src/test/java/org/sonatype/nexus/repository/rest/internal/resources/SearchResourceTest.java [hunk 1]
```diff
@@ -480,7 +480,7 @@ public void testGetAssetParams() {
     Set<String> allKeys = searchUtils.getSearchParameters().keySet();
     allKeys.forEach(s -> sb.append(s).append("=valueDoesNotMatter&"));
 
-    // asert only assert params remain
+    // assert only assert params remain
```
Comment (joedragons) [inline components/nexus-repository/src/test/java/org/sonatype/nexus/repository/rest/internal/resources/SearchResourceTest.java]: Should this second "assert" be "asset"?

Comment (jsoref) [reply components/nexus-repository/src/test/java/org/sonatype/nexus/repository/rest/internal/resources/SearchResourceTest.java]: I'm really not sure. That one definitely puzzled me a bit. Thinking about it again, you may be right.

**Comment by nblair** [comment]
Thank you for the contribution @jsoref -  this is a large change with a lot of conflicts to resolve. I'm going to close this request.

**Comment by jsoref** [comment]
@nblair I'm fully capable of regenerating this work if you're open to reviewing it with a turnaround faster than 3 years...

**Comment by nblair** [comment]
Thanks for the offer @jsoref but that won't be necessary. There's a lot of additional review and test requirements a change like this would require that will quickly test the limits of capacity we have.

**Comment by jsoref** [comment]
I mean the refresh is already done https://github.com/check-spelling/nexus-public/pull/1 and I'm currently running the tests using oracle jdk 1.8...

I'm more than happy to give you a subset, e.g. only user facing things (.jsx?)

**Pull request closed without merging**